src/main/java/com/griddynamics/cloud/learning/GlobalControllerAdvice.java

```diff
@@ -16,4 +17,10 @@
         log.info(ex.getMessage(), ex);
         return new ResponseEntity<>(new ErrorResponse(ex.getMessage()), HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(UserNotFoundException.class)
+    public final ResponseEntity<ErrorResponse> handleUserNotFound(Exception ex) {
+        log.info(ex.getMessage(), ex);
```

Comment:
logic are pretty same. let's move out it to a private method. 
ResponseEntity<ErrorResponse> processException(Exception ex, HttpStatus status)